src/UserForm.test.js

```diff
@@ -0,0 +1,124 @@
+import '__mock__/';
+import '__mock__/reactFinalForm.mock';
```

Comment:
Could you import mock data from `__mock__/reactFinalForm.mock` to `__mock__/index`?
And then here we will have the following view

```suggestion
import '__mock__/';
```